<?xml version="1.0"?>

<launch>
  <arg name="test" default="true"/>
  <arg name="images_path" default=""/>
  <node name="transferability_vision_system" type="TransferabilityVisionSystem.py" pkg="robothon2023_vision" output="screen">
    <param name="test" value="$(arg test)" />
    <param name="images_path" type="string" value="$(find robothon2023_vision)/file/" />
    <param name="weight_path_circuit" type="string" value="$(find robothon2023_vision)/weight/best_ckpt_circuit_noAugment_noPreproc.pt" />
    <param name="labels_path_circuit" type="string" value="$(find robothon2023_vision)/config/circuit.yaml" />
    <param name="weight_path_digits" type="string" value="$(find robothon2023_vision)/weight/best_ckpt_digits_augment_preproc.pt" />
    <param name="labels_path_digits" type="string" value="$(find robothon2023_vision)/config/digits.yaml" />
  </node>
</launch>
